protoc-gen-swagger/defs.bzl

```diff
@@ -99,6 +131,10 @@ protoc_gen_swagger = rule(
             allow_single_file = True,
             mandatory = False,
         ),
+        "allow_merge": attr.bool(
```

Comment:
is `allow_merge` the best name for this? `allow_merge` makes me think that it is possible but not necessary to merge the output files. What would you think about `merge_files` or `single_output`?

Reply:
The rationale here is that is the name of the flag provided to the protoc-gen-swagger.

I don't feel very strongly about this either way, but let me know what you think if it makes sense to match the flag name or use a more idiomatic name.